app/experimenter/experiments/constants/legacy.py

```diff
@@ -1117,6 +1119,8 @@ def FEATURE_TYPE_CHOICES(cls):  # pragma: no cover
         "Experimenter enrollment ending verification " "for: {name} {version} {channel}"
     )
 
+    COMMENT_EMAIL_SUBJECT = "[Experimenter]: {email} commented on {name}"
```

Comment:
You could just pass the experiment in here and do `{experiment.owner.email} commented on {experiment.name}` so you just have to pass the experiment in